theme/static/css/hydroshare_core.css

```diff
@@ -46,42 +46,67 @@ h1.hl small, h2.hl small, h3.hl small, h4.hl small, h5.hl small, h6.hl small {
 .navbar-inverse .navbar-nav > .active > a,
 .navbar-inverse .navbar-nav > .active > a:hover,
 .navbar-inverse .navbar-nav > .active > a:focus {
-  color: #000;
-  font-weight: bold !important;
-  background-color: #eaeaea !important;
+  	color: #333;
+    background-color: #FFF !important;
 }
+
 .navbar-inverse .navbar-brand {
-  color: #EEEEEE;
+	color: #EEEEEE;
 }
+
 .navbar-inverse .navbar-nav > li > a {
-  color: #000;
+	color: #777;
+    font: 13px/18px 'Open Sans',arial,sans-serif;
 }
 
-.navbar-inverse .navbar-nav>li>a:hover, .navbar-inverse .navbar-nav>li>a:focus {
-    color: #000;
-    background-color: #EEEEEE;
+.navbar-nav > li > .dropdown-menu {
+	margin-top: 18px;
+	margin-right: -12px;
+	width: 300px;
+	height: 160px;
+	padding: 0;
+	background: #FFF;
+	border: 1px solid #909090;
+	-webkit-border-radius: 4px;
+	-moz-border-radius: 4px;
+	border-radius: 4px;
+	box-shadow: 1px 1px 5px rgba(0, 0, 0, 0.6);
+	-moz-box-shadow: 1px 1px 5px rgba(0, 0, 0, 0.6);
+	-webkit-box-shadow: 1px 1px 5px rgba(0, 0, 0, 0.6);
+}
+
+.navbar-inverse .navbar-nav > li > a:hover, .navbar-inverse .navbar-nav > li > a:focus {
+	color: #333;
+	-webkit-transition: color 100ms linear;
+	-moz-transition: color 100ms linear;
+	-o-transition: color 100ms linear;
+	-ms-transition: color 100ms linear;
+	transition: color 100ms linear;
 }
+
 .navbar-inverse {
-    font-family: 'Lato', sans-serif;
     text-transform: uppercase;
-    font-size: 12px;
-    border-bottom:1px solid #DDD;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.2);
```

Comment:
You should probably add the fail-safe `-moz-box-shadow` and `-webkit-box-shadow` properties here as well.